cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java

```diff
@@ -226,13 +253,16 @@ private void initializeMessageIds() {
     }
 
     TransactionRunners.run(transactionRunner, context -> {
-      AppMetadataStore appMetadataStore = AppMetadataStore.create(context);
+      AppMetadataStore appMetadataStore = getAppMetadataStore(context);
```

Comment:
shouldn't need to do this anymore, it's handled by superclass and loadMessageId().

Reply:
see previous comment about lastMessageIds